README.md

```diff
@@ -37,29 +56,35 @@ Test that printers are detected using lpstat -v
 Support for both Toshiba and Squix printers. When it receives a request, PMB looks at the printer name it is passed in the request, and determines whether it's a Toshiba or a Squix (`printers` database table, `printer_type` column). It then handles the printing itself for Toshibas, or sends a request to SPrint (https://github.com/sanger/sprint) for Squixes.
 The same templates (`label_templates` and `label_types` tables) can be used with both types of printer.
 
-Print job POST request bodies will look something like this:
+More complete details can be found on the [Confluence Article - Understanding Label Printing](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Understanding+Label+Printing), under the heading _PMB v2_.
 
-```
+#### Print Job Example
+
+To submit a print job, send a POST request to the `/v2/print_jobs` endpoint.  
+The headers should include `Content-Type: application/vnd.api+json`.
+
+The request body should be a JSON object with the following structure:
+
+```json
 {
   "print_job": {
     "printer_name": "stub",
```

Comment:
NB. `stub` is a special printer name in SPrint which has a print protocol adapter to send the received print commands to logging. The logs can be viewed on the SPrint host using `journalctl`. Maybe this can be mentioned in the comment. This suggestion is completely optional and it can be ignored for this PR.